src/Webapi/Webapi__Dom/Webapi__Dom__HtmlDocument.re

```diff
@@ -3,11 +3,13 @@ module Impl = (T: {type t;}) => {
 
   [@bs.get] [@bs.return nullable] external activeElement : t_htmlDocument => option(Dom.element) = "";
   [@bs.get] [@bs.return nullable] external body : t_htmlDocument => option(Dom.element) = ""; /* returns option HTMLBodyElement */
+  [@bs.get] external characterSet : t_htmlDocument => string = "";
```

Comment:
Why has this been moved here from `Document`? The documentation you refer to lists it as a property of `Document`, and `as HtmlDocument` inherits everything from `Document` it should be available here anyway.

Reply:
weird, didnt find it. but will move it here.
In general i wonder what's the actal distinction between document and htmldocument.
In the docs it means that for all porpuses its the same thing when dealing with the web.

Reply:
You have also removed it from `Document`, and from its test, so it is actually moved, not added. Seems like it took you a few iterations to get here though, so I guess a mixup happened somewhere.

Reply:
`HTMLDOcument` is a subtype of `Document`, ie. more specific. In addition to HTML Documents a `Document` can also be an `XMLDocument` and I think also an `SVGDocument`. These kinds of documents also exist on the web.